.github/ISSUE_TEMPLATE/problem.yaml

```
@@ -0,0 +1,63 @@
+---
+name: Problem
+description: Track anything that is broken, wrong or needs fixing
+labels:
+  - problem
+projects:
+  - UCL-MIRSG/3
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Add any new problem - this may be a technical bug, service incident,
+        process problem or general issue that needs to be resolved.
+  - type: textarea
+    id: short_description
+    attributes:
+      label: Short Description
+      description: Briefly summarise the problem.
+      placeholder: What is the problem? What is the impact?
+    validations:
+      required: true
+  - type: textarea
+    id: evidence
+    attributes:
+      label: Evidence/Steps to Reproduce
+      description: |
+        Can you reproduce the problem? If so, what are the steps to reproduce it?
+        Otherwise, provide any evidence that helps to understand the problem.
+      placeholder: e.g. List of steps, screenshots, logs, error messages, etc.
+    validations:
+      required: false
+  - type: textarea
+    id: acceptance_criteria
+    attributes:
+      label: Acceptance Criteria/Expected Behaviour
+      description: |
+        What is the expected behaviour once the problem is resolved?
+        How will you know when the problem is fixed?
+      placeholder: e.g. "When the problem is resolved, the system should do x, y, z"
+    validations:
+      required: false
+  - type: textarea
+    id: details
+    attributes:
+      label: Details
+      description: Provide any additional information that may be useful
+      placeholder: e.g. Additional context, links to documentation and diagrams, etc.
+  - type: markdown
+    attributes:
+      value: |
+        Thanks you for raising this problem!
+        ---
+        Fill in the below once the problem is resolved:
+  - type: textarea
+    id: resolution
+    attributes:
+      label: Resolution
+      description: |
+        What was done to resolve the problem?
+        What was the root cause? Reference any related issues or pull requests.
+      placeholder: "e.g. 'The problem was resolved by doing x, y, z in #123'"
```

Consistent with elsewhere
```suggestion
      placeholder: e.g. "The problem was resolved by doing x, y, z in #123"
```